main/constants/constants.ts

```diff
@@ -1,6 +1,6 @@
-export const DI_ROOT_INJECTOR_KEY = '__DI__ROOT__INJECTOR__';
-export const DI_GLOBAL_STATE_STORE = '__DI__GLOBAL_STATE_STORE';
-export const GLOBAL_INSTANCE_MAP = 'GLOBAL_INSTANCE_MAP';
-export const GLOBAL_REGISTRATION_MAP = 'GLOBAL_REGISTRATION_MAP';
-export const GLOBAL_CONFIGURATION = 'GLOBAL_CONFIGURATION';
+export const DI_ROOT_INJECTOR_KEY = 'morganstanley.com.root.injector';
```

Comment:
I would also have added a project level namespace
`morganstanley.com.needle.root.injector`
to match our npm name to ensure no conflicts.

Reply:
I'll update.